README.md

```diff
@@ -228,45 +228,45 @@ npm run prepublishOnly
+| [`15.d.backchannel-send-welcome-event`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/15.d.backchannel-send-welcome-event)                                                                                 | Advanced tutorial: Demonstrates how to send welcome event with client capabilities such as browser language.                                                                                                                        | [Welcome Event Demo](https://microsoft.github.io/BotFramework-WebChat/15.d.backchannel-send-welcome-event)                          |
+| [`16.customization-selectable-activity`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/16.customization-selectable-activity)                                                                               | Advanced tutorial: Demonstrates how to add custom click behavior to each activity.                                                                                                                                                  | [Selectable Activity Demo](https://microsoft.github.io/BotFramework-WebChat/16.customization-selectable-activity)                   |
+| [`17.chat-send-history`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/17.chat-send-history)                                                                                                               | Advanced tutorial: Demonstrates the ability to save user input and allow the user to step back through previous sent messages.                                                                                                      | [Chat Send History Demo](https://microsoft.github.io/BotFramework-WebChat/17.chat-send-history)                                     |
+| [`18.customization-open-url`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/18.customization-open-url)                                                                                                     | Advanced tutorial: Demonstrates how to customize the open URL behavior.                                                                                                                                                             | [Customize Open URL Demo](https://microsoft.github.io/BotFramework-WebChat/18.customization-open-url)                               |
+| [`19.customization-plain-ui`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/19.customization-plain-ui)                                                                                                     | Advanced tutorial: Demonstrates how to customize Web Chat UI by building from ground up.                                                                                                                                            | [Customize with Plain UI](https://microsoft.github.io/BotFramework-WebChat/19.customization-plain-ui)                               |
```

Comment:
```suggestion
| [`19.customization-plain-ui`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/19.customization-plain-ui)                                                                                                     | Advanced tutorial: Demonstrates how to customize the Web Chat UI by building from ground up instead of needing to rewrite entire Web Chat components.                                                                                                                                       | [Plain UI Demo](https://microsoft.github.io/BotFramework-WebChat/19.customization-plain-ui)                               |
```